packages/hoppscotch-common/src/components/http/RequestOptions.vue

```diff
@@ -72,6 +72,10 @@ const emit = defineEmits<{
 
 const request = ref(props.modelValue)
 
+watch(props, (updatedProps) => {
```

Comment:
I think we have `request` just representing the modelValue right ?
In that case, why not just define `request` using `useVModel` ?

Will something break ?
```ts
import { useVModel } from "@vueuse/core"

const request = useVModel(props, "modelValue", emit)
```